tst/integration/error_handlers.py

```diff
@@ -15,14 +15,14 @@ def is_wrongtype_error(string):
     def is_number_overflow_error(string):
         return string.startswith("OVERFLOW")
 
-    def is_outofboundaries_error(string):
-        return string.startswith("OUTOFBOUNDARIES")
+    def is_outofbounds_error(string):
+        return string.startswith("OUTOFBOUNDS")
 
     def is_limit_exceeded_error(string):
         return string.startswith("LIMIT")
 
     def is_write_error(string):
-        return string.startswith("ERROR") or string.startswith("OUTOFBOUNDARIES") or \
+        return string.startswith("ERROR") or string.startswith("OUTOFBOUNDS") or \
```

Comment:
These changes indicate that I'm changing something of an API. If that's a problem, I can drop the changes...